tests/e2e-food.json

```diff
@@ -266,38 +271,39 @@
       "wtu": 0.09704827341756544
     },
     "scoring": {
-      "category": "Toutes les catégories",
+      "category": "Fruits et légumes",
       "all": {
         "impact": 1230.7521714388874,
         "letter": "E",
-        "outOf100": 15
+        "outOf100": 0
```

Comment:
Not sure about the computation. I should have missed something. The formula seems ok but If I apply for 1230 I get a score of 17, not 0

```
> ln = logBase e
> impact0=2270
> impact100=70.9
> score impact= 100 * (ln impact0 - ln impact) / (ln impact0 - ln impact100)
> score 1230
17.677982474506152 : Float
```

Reply:
You missed the `"category=cakes"` param set, which will use the `cakes` category bounds, which are `{ impact100 = 100, impact0 = 700 }`.